README.md

```diff
@@ -891,7 +894,43 @@ Now the last required step is to register the `MusicianSerializer` to the config
 
 From now on, Hazelcast will use `MusicianSerializer` to serialize `Musician` objects.
 
-## 4.4. Global Serialization
+## 4.4. JSON Serialization
+
+You can use JSON formatted strings as objects in Hazelcast cluster. Starting with Hazelcast server version 3.12, the JSON serialization is one of the formerly supported serialization methods. Creating JSON objects in the cluster does not require any server side coding and hence you can just send a JSOn formatted string object to the cluster and query these objects by fields.
+
+In order to use JSON serialization, you should use `HazelcastJsonValue` object for the key or value. Here is an example IMap usage:
+
+```C++
+    hazelcast::client::ClientConfig config;
+    hazelcast::client::HazelcastClient hz(config);
+
+    hazelcast::client::IMap<std::string, serialization::json::HazelcastJsonValue> map = hz.getMap<std::string, serialization::json::HazelcastJsonValue>(
+            "map");
+```
+
+We constructed a map in the cluster which has the `string` as the key and `HazelcastJsonValue` as the value. `HazelcastJsonValue` is a simple wrapper and identifier for the JSON formatted strings. You can get the JSON string from the `HazelcastJsonValue` object using the `toJsonString()` nethod. 
+
+You can construct the `HazelcastJsonValue` using one of the constructors. All constructors accept the JSON formatted string as the parameter. No JSON parsing is performed but it is your responsibility to provide correctly formatted JSON strings. The client will not validate the string, and it will send it to the cluster as it is. If you submit incorrect formatted JSON strings and later if you query those objects it is highly possible that you will get formatting errors since the server will fail to deserialize or find the query fields.
```

Comment:
... If you submit incorrectly formatted JSON strings and, later, if you query those objects, it is highly ...